IAMOTZ/More-recipes
# #152082541 API for modifying recipes

#### What should this PR do?
It should enable modifying of of recipes stored in a dummy database.

#### Description of Task to be completed?
User modify a recipe in his/her catalog(API endpoint).
#### How should this be manually tested?

- clone the repo
- cd into the server directory
- run the following commands
- `npm install` to install dependencies
- `npm run start:dev` to start the server on port 3000
- On postman or any API testing tool, send put request to `/api/recipes/<recipeId>`
- The post request should be accompanied by a body containing: 
`{ title: "", body: ""}`
- Fix in whatever you like in the id, title and body properties
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#152082541
#### Screenshots (if appropriate)
None
#### Questions:
None
